src/Resources/ResourceManager/Properties/Resources.resx

```diff
@@ -144,6 +144,9 @@
   <data name="DeploymentDoesntExistAtSubscriptionScope" xml:space="preserve">
     <value>Deployment '{0}' doesn't exist in subscription.</value>
```

Comment:
>subscription [](start = 45, length = 12)

You didn't change this, but should we include the subscription ID in the error?

Reply:
For all the subscription level scenarios, we work on the current subscription in the context. same for tenant. We could put the subscription Id in the message, but my feeling is it's not necessary. Also, I'm not sure how to get the current tenant Id. :)

Reply:
Ok, sounds good.

---
In reply to: [307948665](https://github.com/Azure/azure-powershell/pull/9707#discussion_r307948665) [](ancestors = 307948665)